workos/types/events/directory_payload_with_legacy_fields.py

```diff
@@ -5,7 +5,7 @@
 
 class MinimalOrganizationDomain(WorkOSModel):
     id: str
-    organization_id: str
+    domain: str
```

Comment:
This is technically a breaking change.

Reply:
Is that because we are going to start returning correct objects instead of dicts?

Reply:
A key that, right now, is always `None`.

What I'm thinking about is this: since this type is wrong, it's always parsing the Webhook incorrectly, returning a `dict`.

The behavior is worse if they try to retrieve Events. Those don't have a `dict` fallback mechanism, because the client needs to explicitly send the list of event types they want. I think retrieving this type of Event flat-out error because it fails to parse.

If I'm correct about this above, this seems like a more urgent bug fix to me.

Reply:
Talked about this in DMs but bringing the convo here, I think this is unfortunately going to require a major version release. Anyone who is currently using the SDK for webhooks and not listing events is going to have their integration broken if we push this out as a minor or patch.